app/grants/tasks.py

```diff
@@ -197,6 +197,9 @@ def process_grant_contribution(self, grant_id, grant_slug, profile_id, package,
 
         # emails to grant owner
         new_supporter(grant, subscription)
+
+        # emails to contributor
+        thank_you_for_supporting([grant.pk], subscription.profile)
```

Comment:
this came from my commit ! looks like rebase wasn't done right 😓

Reply:
conflict resolved!